cities/cities.json

```diff
@@ -87,10 +87,6 @@
         "id": "hilden",
         "label": "Hilden"
     },
-    "kaiserslautern": {
```

Comment:
Please undo all changes to the entries of other cites in all files. Take a look at the "Files changed" tab here on this website - it shows you the "sum" of all commits. Please do not change the text of Zwickau. You can also preview the changes in the README.md if you just visit your `master` branch [here](https://github.com/operator2010/wo-ist-markt.github.io).